rfc5425/parser.go

```diff
@@ -0,0 +1,137 @@
+package rfc5425
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/influxdata/go-syslog/rfc5424"
+)
+
+// Parser represents a parser
+type Parser struct {
+	s          Scanner
+	p          rfc5424.Parser
+	bestEffort bool
+	msglen     int64
+	buf        struct {
+		tok Token // last read token
+		num int   // size (max = 1)
+	}
+}
+
+// NewParser returns a new instance of Parser
+func NewParser(r io.Reader, bestEffort bool) *Parser {
```

Comment:
I really don't like bool parameters... looks nice from the library but not the library user

```go
NewParser(r, true)
```
is hard to know what is going on.

I think you should use an opts pattern similar to what I did with telegraf.